src/stores/MessageStore.js

```diff
@@ -9,17 +9,6 @@ let CHANGE_EVENT = 'change';
 
 let _messages = {};
 
-function _addMessages(rawMessages) {
```

Comment:
@AnshulMalik Every Message is being mapped to a  _messages array and this is also being used to update the latest timestamp if you look at the function it is calling ie. ChatMessageUtils.convertRawMessage()
```
export function convertRawMessage(rawMessage, currentThreadID) {
  return {
    ...rawMessage,
    date: new Date(rawMessage.timestamp),
    isRead: rawMessage.threadID === currentThreadID
  };
};
```
Also when i tried deleting it the chat messages weren't being updated on the chat window, which is not surprising  at all because of course it needs to be added to the array of messages right.